src/oc/storage/resources/entry.clj

```diff
@@ -89,21 +90,21 @@
   "
   Given the ID of the entry, retrieve the entry, or return nil if it doesn't exist.
 
-  Or given the UUID of the org or board, a topic slug, and the created-at timestampt,
+  Or given the UUID of the org or board, a topic slug, and the created-at timestamp,
   retrieve the entry, or return nil if it doesn't exist. In this case a keyword, `:org` or `:board`
   needs to be provided to indicate what type of UUID is being used.
   "
-  ([conn id :- lib-schema/UUIDStr]
+  ([conn uuid :- lib-schema/UniqueID]
   {:pre [(db-common/conn? conn)]}
-  (db-common/read-resource conn table-name id))
+  (db-common/read-resource conn table-name uuid))
+
+  ([conn org-uuid :- lib-schema/UniqueID board-uuid :- lib-schema/UniqueID topic-slug :- common/TopicSlug uuid :- lib-schema/UniqueID]
+  {:pre [(db-common/conn? conn)]}
+  (first (db-common/read-resources conn table-name :uuid-topic-slug-board-uuid-org-uuid [[uuid topic-slug board-uuid org-uuid]])))
   
-  ([conn uuid-type :- (schema/enum :board :org) uuid :- lib-schema/UniqueID
-    topic-slug :- common/TopicSlug created-at :- lib-schema/ISO8601]
+  ([conn org-uuid :- lib-schema/UniqueID topic-slug :- common/TopicSlug created-at :- lib-schema/ISO8601]
```

Comment:
Why before we had entries connected to board uuid or org uuid and now we have them only by org?
Is this change needed by the new entry schema?